gabrielaEnache/src/main/java/collections/utils/Generator.java

```diff
@@ -0,0 +1,9 @@
+package collections.utils;
+
+public class Generator {
+    private static long generatedId = 0;
+
+    public static long generateId() {
```

Comment:
Nice try...This means that any time you will call this function it will make ++ID. In real word there are situation when Order id = 0 and User id = 0, by using this method to generate ID you will be never able to do it.)

Reply:
fixed